src/bot/commands/general/transfer.ts

```diff
@@ -0,0 +1,44 @@
+import { Command, CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { Message } from 'discord.js';
+import createPlayer from '../../../utils/create-player';
+import { isBound } from '../../../utils/isBound';
+import transferEvent from '../../../actions/transfer';
+
+export default class DisbandCommand extends Command {
+  public constructor(client: CommandoClient) {
+    super(client, {
+      name: 'transfer',
+      memberName: 'transfer',
+      group: 'general',
+      description: 'Transfer group ownership to another player',
+    });
+  }
+
+  public async run(msg: CommandoMessage): Promise<Message> {
+    const bindResult = await isBound(msg);
+    if (!bindResult) {
+      return msg.channel.send(
+        'This bot is not yet configured. You must first bind the bot to a channel with `!valorant bind`'
+      );
+    }
+
+    await createPlayer({
+      name: msg.member.displayName,
+      discordId: msg.member.id,
+      rank: 'iron1',
+    });
+
+    const targetId = msg.content
+      .split(' ')
+      .slice(-1)[0]
+      .replace('<@!', '')
+      .replace('>', '');
```

Comment:
You can get the mentions straight out of the msg object instead of parsing it this way. Parsing the string like this is very fragile if discord were to ever change anything.

see: https://discord.js.org/#/docs/main/stable/class/Message?scrollTo=mentions